js/templates/modals/wallet/wallet.html

```diff
@@ -26,10 +26,10 @@
           </div>
         </div>
         <div class="clrP clrSh3 posR">
-          <div class="js-reloadTransactionsContainer reloadTransactions"></div>
           <div class="js-transactionsContainer"></div>
+          <div class="js-reloadTransactionsContainer reloadTransactions"></div>
```

Comment:
Since this div is absolutely positioned, it needs to be after the transactionsContainer div, so it will be layered on top of it.